### PR TITLE
Made sentence less wordy and fixed typo

### PR DESCRIPTION
**Pull request by JoelMon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ language that you can read [here](https://rust-unofficial.github.io/patterns/).
 
 ## Contributing
 
-You are missing content in this repository that can be helpful for others and
-you are eager to explain it? Awesome! We are always happy about new contributions
+Are you eager to eager to contribute missing content?
```
Comment (pickfire) [inline README.md:8]: I think it might still be better to have "helpful for others", seemed like a plus to keep that.

**Comment by JoelMon** [comment]
Clicked on "resolve conversation" without being sure what it did. I will send a new PR for this one.

File: README.md (hunk 1 above)
Comment (JoelMon) [reply README.md:8]: > Are you eager to eager to contribute missing content _that is helpful for others_?

**Comment by simonsan** [comment]
> Clicked on "resolve conversation" without being sure what it did. I will send a new PR for this one.

No problem, unresolved it. :)

**Pull request closed without merging**